Cargo.toml

```diff
@@ -14,6 +14,7 @@ members = [
     "src/expr",
     "src/interchange",
     "src/kafka-util",
+    "src/lint",
```

Comment:
A slightly more specific name than lint would be ideal! Like "sql-lint" or "expr-lint" or somesuch.